wouso/games/teamquest/models.py

```diff
@@ -95,6 +95,15 @@ def __init__(self, *args, **kwargs):
         # TODO
         super(TeamQuestGame, self).__init__(*args, **kwargs)
 
+    @classmethod
+    def get_current(cls):
+
```

Comment:
Replace this empty line with a docstring. It's not clear what it does without looking at the code